templates.py

```diff
@@ -0,0 +1,31 @@
+submission_exceeding_threshold: str = '''
+All apologies /u/{} but your post has been automatically removed because the artist has too many {}. The maximum is {:,}, this link has {:,}.
+If you think this is in error, please [contact the mods](https://www.reddit.com/message/compose?to=/r/listentothis&subject=Post+removed+in+error.&message=https://redd.it/{}). 
+
+If you're new to the subreddit, please read the full list of removal reasons.
+
+_Don't blame me,_ [_I'm just a bot_](https://www.youtube.com/watch?v=jqaweMZv4Og)|[_Bugs & Code_](https://github.com/martijnboers/BlotterTrax)
+        '''
+submission_missing_discussion_tag: str = '''
```

Comment:
Worth adding some additional whitespace in this file just to make it easier to spot where a new template begins in apps that don't do great color highlighting (for instance, this Github PR diff UI!)